integration/copy_test.go

```diff
@@ -577,7 +586,7 @@ func (s *CopySuite) TestCopyEncryption(c *check.C) {
 		"oci:"+encryptedImgDir+":encrypted", "oci:"+decryptedImgDir+":decrypted")
 
 	// Copy a standard busybox image locally
-	assertSkopeoSucceeds(c, "", "copy", "docker://busybox", "oci:"+originalImageDir+":latest")
+	assertSkopeoSucceeds(c, "", "copy", "docker://busybox:1.31.1", "oci:"+originalImageDir+":latest")
```

Comment:
Why this change? (It’s not obviously _undesirable_, but it seems quite unrelated and I can’t see why it is necessary.)

Reply:
To accomodate partial image encryption, I changed existing test case for image encryption slightly. So now after encrypting the image we count how many layers in the target image got encrypted. If you do a full image encryption then the number of layers from the original image should match with the number of encrypted layers in the target image. While if you do partial image encryption, the number of encrypted layers in the target image should be equivalent to the number of `encrypt-layer` argument(s) while encrypting the original image. 

So the reason I replaced `latest` with `1.31.1` is to make sure our test case shouldn't fail if the number of layers in the busybox change in future.

Reply:
Thanks!